gnomad/assessment/sanity_checks.py

```diff
@@ -103,90 +117,832 @@ def make_filters_sanity_check_expr(
+    problematic_regions: List[str] = ["lcr", "segdup", "nonpar"],
+    single_filter_count: bool = False,
+    summarize_variants_check: bool = True,
+    filters_check: bool = True,
+    raw_adj_check: bool = True,
+    subset_freq_check: bool = True,
+    samples_sum_check: bool = True,
+    sex_chr_check: bool = True,
+    missingness_check: bool = True,
+) -> None:
+    """
+    Perform a battery of sanity checks on a specified group of subsets in a MatrixTable containing variant annotations.
+
+    Includes:
+    - Summaries of % filter status for different partitions of variants
+    - Histogram outlier bin checks
+    - Checks on AC, AN, and AF annotations
+    - Checks that subgroup annotation values add up to the supergroup annotation values
+    - Checks on sex-chromosome annotations; and summaries of % missingness in variant annotations
+
+    All annotations must be within an info struct, e.g. t.info.AC-raw.
+
+    :param t: Input MatrixTable or Table containing variant annotations to check.
+    :param subsets: List of subsets to be checked.
+    :param pops: List of pops within main callset.
+    :param missingness_threshold: Upper cutoff for allowed amount of missingness. Default is 0.5.
+    :param monoallelic_expr: When passed, log how many monoallelic sites are in the Table.
+    :param verbose: If True, display top values of relevant annotations being checked, regardless of whether check conditions are violated; if False, display only top values of relevant annotations if check conditions are violated.
+    :param show_percent_sites: Show percentage of sites that fail checks. Default is False.
+    :param delimiter: String to use as delimiter when making group label combinations. Default is "-".
+    :param metric_first_label: If True, metric precedes label group, e.g. AC-afr-male. If False, label group precedes metric, afr-male-AC. Default is True.
+    :param sum_metrics: List of metrics to sum and compare to annotationed versions and between subsets and entire callset. Default is ["AC", "AN", "nhomalt"].
+    :param sexes: List of sexes in table. Default is SEXES.
+    :param sample_sum_sets_and_pops: Dict with subset (keys) and populations within subset (values) for sample sum check.
+    :param sort_order: List containing order to sort label group combinations. Default is SORT_ORDER.
+    :param variant_filter_field: String of variant filtration used in the filters annotation on `ht` (e.g. RF, VQSR, AS_VQSR). Default is "RF".
+    :param problematic_regions: List of regions considered problematic to run filter check in. Default is ["lcr", "segdup", "nonpar"].
+    :param single_filter_count: If True, explode the Table's filter column and give a supplement total count of each filter. Default is False.
+    :param summarize_variants_check: When true, runs the summarize_variants method. Default is True.
+    :param filters_check: When true, runs the summarize_variant_filters method. Default is True.
```

Comment:
```suggestion
    :param filters_check: When True, runs the summarize_variant_filters method. Default is True.
```
(just for consistency with the above `If True`s)